app/src/main/java/eu/kanade/tachiyomi/App.kt

```diff
@@ -196,6 +199,19 @@ class App : Application(), DefaultLifecycleObserver, SingletonImageLoader.Factor
                 add(MangaKeyer())
             }
 
+            memoryCache(
+                MemoryCache.Builder()
+                    .maxSizePercent(context, 0.25)
```

Comment:
```suggestion
                    .maxSizePercent(context)
```
1. https://coil-kt.github.io/coil/api/coil-core/coil3.memory/-memory-cache/-builder/index.html#-1971330947%2FFunctions%2F1448608982
2. https://github.com/coil-kt/coil/blob/46130609cc03bd5313bd7d902552e372225c3783/coil-core/src/androidMain/kotlin/coil3/util/contexts.kt#L30-L37